applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/app_ZReleaseMCUBoot.conf

```diff
@@ -6,10 +6,13 @@
 ################################################################################
 # Application Configuration
 
+CONFIG_CAF=y
```

Comment:
Just a small nit.
Maybe move this line a little down, so that all `CAF` settings are grouped together like:
```
CONFIG_CAF=y
CONFIG_CAF_BUTTONS_ENABLE=y
CONFIG_CAF_BUTTONS_PM_EVENTS_ENABLE=y
CONFIG_CAF_BUTTONS_POLARITY_INVERSED=y
```

This applies to all conf files.

Reply:
Will do. Just need a bit of a clean up rn.